NWNXLib/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ macro(nwnxlib_add)
     endif()
 endmacro()
 
-nwnxlib_add("Assert.cpp" "Log.cpp" "Plugin.cpp" "Serialize.cpp" "Utils.cpp" "Encoding.cpp")
+nwnxlib_add("Assert.cpp" "Log.cpp" "Plugin.cpp" "Serialize.cpp" "Utils.cpp" "Encoding.cpp" NWVariant.cpp)
```

Comment:
Bikeshed: I find the name confusing. I'd like to treat anything `C[A-Z]\w+` or `NW\w+` as base game reserved. I understand this is in NWNXLib namespace, but most code is `using` that already.

How about just `Variant`? Or `GameType`, `ScriptType`, since it holds any of the times visible to nwscript? `ScriptVariant` as a combo? etc..

Reply:
I didn't really like the name either.  I like `ScriptVariant`, I think that's the clearest of all the options.